conftest.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
-# ----------------------------------------------------------------------------
-# Copyright (C) 2013-2019 British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Lesser General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-# ----------------------------------------------------------------------------
-
-"""Local conftest.py plugins contain directory-specific hook implementations.
-Session and test running activities will invoke all hooks defined in
-conftest.py files closer to the root of the filesystem."""
-
-import pytest
-
-
-def pytest_addoption(parser):
-    parser.addoption(
-        "--runslow", action="store_true", default=False, help="run slow tests"
```

Comment:
pytest has a built-in system for marking tests so we don't have to implement our own.